[🐛 Bug]: Grid GraphQL query failure: Unable to parse

### What happened?

Trying to use the graphQL query example [from the documentation](https://www.selenium.dev/documentation/grid/advanced_features/graphql_support/#query-to-get-session-information-for-a-given-session) for session info results in an "Unable to parse" error.


### How can we reproduce the issue?

```shell
curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: "<session-id>") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
```


### Relevant log output

```shell
{
  "value": {
    "error": "unknown error",
    "message": "Unable to parse: {\"query\":\"{ session (id: \"8675309\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } \"}",
    "stacktrace": "org.openqa.selenium.json.JsonException: Unable to parse: {\"query\":\"{ session (id: \"8675309\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } \"}\r\n\tat org.openqa.selenium.json.Json.toType(Json.java:57)\r\n\tat org.openqa.selenium.json.Json.toType(Json.java:50)\r\n\tat org.openqa.selenium.grid.graphql.GraphqlHandler.execute(GraphqlHandler.java:114)\r\n\tat org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle(Route.java:192)\r\n\tat org.openqa.selenium.remote.http.Route.execute(Route.java:68)\r\n\tat org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)\r\n\tat org.openqa.selenium.remote.http.Route.execute(Route.java:68)\r\n\tat org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)\r\n\tat org.openqa.selenium.remote.http.Route.execute(Route.java:68)\r\n\tat org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0(AddWebDriverSpecHeaders.java:35)\r\n\tat org.openqa.selenium.remote.ErrorFilter.lambda$apply$0(ErrorFilter.java:44)\r\n\tat org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)\r\n\tat org.openqa.selenium.remote.ErrorFilter.lambda$apply$0(ErrorFilter.java:44)\r\n\tat org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)\r\n\tat org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0(SeleniumHandler.java:44)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)\r\n\tat java.util.concurrent.FutureTask.run(Unknown Source)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)\r\n\tat java.lang.Thread.run(Unknown Source)\r\nCaused by: org.openqa.selenium.json.JsonException: Unable to read name. Expected colon separator, but saw '}'\nBuild info: version: '4.1.1', revision: 'e8fcc2cecf'\nSystem info: host: 'LEWVDTAUTOHUB4', ip: '10.120.20.159', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_271'\nDriver info: driver.version: unknown\r\n\tat org.openqa.selenium.json.JsonInput.nextName(JsonInput.java:142)\r\n\tat org.openqa.selenium.json.StringCoercer.lambda$apply$0(StringCoercer.java:38)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.lambda$buildCoercer$6(JsonTypeCoercer.java:145)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.coerce(JsonTypeCoercer.java:127)\r\n\tat org.openqa.selenium.json.ObjectCoercer.lambda$apply$0(ObjectCoercer.java:72)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.lambda$buildCoercer$6(JsonTypeCoercer.java:145)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.coerce(JsonTypeCoercer.java:127)\r\n\tat org.openqa.selenium.json.MapCoercer.lambda$apply$0(MapCoercer.java:68)\r\n\tat java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)\r\n\tat java.util.Iterator.forEachRemaining(Unknown Source)\r\n\tat java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.copyInto(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)\r\n\tat java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.evaluate(Unknown Source)\r\n\tat java.util.stream.ReferencePipeline.collect(Unknown Source)\r\n\tat org.openqa.selenium.json.MapCoercer.lambda$apply$1(MapCoercer.java:72)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.lambda$buildCoercer$6(JsonTypeCoercer.java:145)\r\n\tat org.openqa.selenium.json.JsonTypeCoercer.coerce(JsonTypeCoercer.java:127)\r\n\tat org.openqa.selenium.json.Json.toType(Json.java:71)\r\n\tat org.openqa.selenium.json.Json.toType(Json.java:55)\r\n\t... 19 more\r\n"
  }
}
```


### Operating System

Windows10

### Selenium version

4.1.0

### What are the browser(s) and version(s) where you see this issue?

n/a

### What are the browser driver(s) and version(s) where you see this issue?

n/a

### Are you using Selenium Grid?

4.1.1

--- a/website_and_docs/content/documentation/grid/advanced_features/graphql_support.en.md
+++ b/website_and_docs/content/documentation/grid/advanced_features/graphql_support.en.md
@@ -107,11 +107,11 @@ The structure of grid schema is as follows:
 
 ## Querying GraphQL
 
-The best way to query GraphQL is by using `curl` requests. GraphQL allows you to 
-fetch only the data that you want, nothing more nothing less.
+The best way to query GraphQL is by using `curl` requests. The query is interpreted as JSON. Ensure double quotes are properly escaped to avoid unexpected errors.
+GraphQL allows you to fetch only the data that you want, nothing more nothing less.
 
 Some of the example GraphQL queries are given below. You can build your own 
-queries as you like.
+queries as you like. 
 
 ### Querying the number of `maxSession` and `sessionCount` in the grid :
 
@@ -154,7 +154,7 @@ curl -X POST -H "Content-Type: application/json" --data '{"query":"{ sessionsInf
 ### Query to get session information for a given session:
 
 ```shell
-curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: "<session-id>") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
+curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: \"<session-id>\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
 ### Querying the capabilities of each node in the grid :

--- a/website_and_docs/content/documentation/grid/advanced_features/graphql_support.ja.md
+++ b/website_and_docs/content/documentation/grid/advanced_features/graphql_support.ja.md
@@ -130,13 +130,13 @@ curl -X POST -H "Content-Type: application/json" --data '{"query":"{ grid { uri,
 curl -X POST -H "Content-Type: application/json" --data '{"query":"{ grid { sessionCount } }"}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
-### グリッドで最大セッション数を取得するためのクエリ 
+### グリッドで最大セッション数を取得するためのクエリ
 
 ```shell
 curl -X POST -H "Content-Type: application/json" --data '{"query":"{ grid { maxSession } }"}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
-### グリッド内のすべてのノードのすべてのセッションの詳細を取得するためのクエリ 
+### グリッド内のすべてのノードのすべてのセッションの詳細を取得するためのクエリ
 
 ```shell
 curl -X POST -H "Content-Type: application/json" --data '{"query":"{ sessionsInfo { sessions { id, capabilities, startTime, uri, nodeId, nodeId, sessionDurationMillis } } }"}' -s <LINK_TO_GRAPHQL_ENDPOINT>
@@ -151,7 +151,7 @@ curl -X POST -H "Content-Type: application/json" --data '{"query":"{ sessionsInf
 ### 特定のセッションのセッション情報を取得するためのクエリ
 
 ```shell
-curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: "<session-id>") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
+curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: \"<session-id>\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
 ### グリッド内の各ノードのcapabilityを照会する

--- a/website_and_docs/content/documentation/grid/advanced_features/graphql_support.pt-br.md
+++ b/website_and_docs/content/documentation/grid/advanced_features/graphql_support.pt-br.md
@@ -150,7 +150,7 @@ curl -X POST -H "Content-Type: application/json" --data '{"query":"{ sessionsInf
 ### Buscando informação da sessão para uma sessão específica:
 
 ```shell
-curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: "<session-id>") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
+curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: \"<session-id>\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
 ### Buscando os recursos de cada nó na Grid:

--- a/website_and_docs/content/documentation/grid/advanced_features/graphql_support.zh-cn.md
+++ b/website_and_docs/content/documentation/grid/advanced_features/graphql_support.zh-cn.md
@@ -151,7 +151,7 @@ curl -X POST -H "Content-Type: application/json" --data '{"query":"{ sessionsInf
 ### 查询以获取给定会话的会话信息查询以获取给定会话的会话信息 :
 
 ```shell
-curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: "<session-id>") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
+curl -X POST -H "Content-Type: application/json" --data '{"query":"{ session (id: \"<session-id>\") { id, capabilities, startTime, uri, nodeId, nodeUri, sessionDurationMillis, slot { id, stereotype, lastStarted } } } "}' -s <LINK_TO_GRAPHQL_ENDPOINT>
 ```
 
 ### 查询网格中每个节点的功能 :

